charts/seed-bootstrap/charts/kube-state-metrics/templates/kube-state-metrics.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-state-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    component: kube-state-metrics
+    type: seed
+
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name: kube-state-metrics
```

Comment:
These global resources might already exist. Can we prefix with `gardener.cloud:monitoring` or something like this?

Reply:
Done